src/components/SlateEditor/plugins/embed/SlateVideo.tsx

```diff
@@ -76,24 +92,42 @@ const SlateVideo = ({
       setStartTime(getStartTime(url));
       setStopTime(getStopTime(url));
     }
-  }, [embed]);
+  }, [tEmbed]);
 
   const toggleEditModus = () => {
     setEditMode(!editMode);
   };
 
+  const switchEmbedSource = () => {
+    const newEmbed = isOriginalEmbed && linkedVideoId ? linkedVideoId : embed.videoid;
+    setEmbed(embed => ({ ...embed, videoid: newEmbed }));
+  };
+
   return (
     <div className="c-figure" draggable="true" {...attributes}>
       <FigureButtons
         tooltip={t('form.video.remove')}
         onRemoveClick={onRemoveClick}
-        embed={embed}
+        embed={tEmbed}
         figureType="video"
-        language={language}
-      />
+        language={language}>
+        <Tooltip
+          tooltip={
+            isOriginalEmbed ? t('form.video.toLinkedVideo') : t('form.video.fromLinkedVideo')
+          }
+          align="right">
+          {linkedVideoId ? (
+            <IconButton as={SafeLink} onClick={switchEmbedSource}>
+              ST
+            </IconButton>
+          ) : (
+            undefined
+          )}
```

Comment:
Noen grunn til at du ikkje gjør slik?
```suggestion
          {linkedVideoId && (
            <IconButton as={SafeLink} onClick={switchEmbedSource}>
              ST
            </IconButton>
          )}
```
Og ST burde vore i i18n.